microprofile-metrics/src/main/java/org/quickstart/microprofile/metrics/JaxRsApplication.java

```diff
@@ -0,0 +1,8 @@
+package org.quickstart.microprofile.metrics;
```

Comment:
package is not correct. It should be `org.wildfly.quickstart...`

Reply:
thank you, I missed this.